src/qps.cpp

```diff
@@ -996,9 +996,9 @@ void Qps::make_command_menu()
     if (flag_devel)
     {
         m_command->addAction( tr( "WatchDog" ), watchdogDialog,
-                             SLOT(show()));
+                             &Qps::show);
```

Comment:
In all places up to this point, you followed what I'd call good practice and used signals/slots of original widgets when they weren't overridden. Please use `&QWidget::show` here for the sake of consistency, in a separate commit and without squashing (for me to continue the review).

Reply:
Done!